src/core/processDirectory.ts

```
@@ -0,0 +1,85 @@
+
+        const summary: DirectorySummary = {
+            errors: 0,
+            files: [],
+            renamed: 0,
+            skipped: 0,
+        };
+
+        // Process all files
+        for (const file of files) {
+            const result = await processFile(dirPath, file, {
+                dateFormat: options.dateFormat as any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)